flow/embedded_views.cc

```diff
@@ -42,13 +42,19 @@ bool DisplayListEmbedderViewSlice::recording_ended() {
   return builder_ == nullptr;
 }
 
-void ExternalViewEmbedder::SubmitFrame(
+void ExternalViewEmbedder::SubmitView(
```

Comment:
I wonder if there's a clearer name for this that isn't overly long -- `SubmitViewFrame` for example?

Reply:
I wonder how much we need to mention "frame" in this context, since of course a view is part of a frame, and the relationship is also explained in the lifecycle described in `ExternalViewEmbedder`'s doc.

If we really need it, maybe `FrameSubmitView` is better, since it shows that the view is part of a frame. What do you think?